### ISSUE TITLE
Omit (-O) giving strange output

### ISSUE THREAD

**Issue by drws** [esnet/iperf#1261]
# Context

* Version of iperf3: 3.10.1

* Hardware: x86_64

* Operating system (and distribution, if any): Linux 5.10

# Bug Report

* Expected Behavior

When using `--omit n`, the first _n_ seconds of test should be omitted in the final result calculation.

* Actual Behavior

Strange things happen. The first is an additional line during runtime output for the (last) omitted interval. The second one is that `--omit` appears, well, _omitted_, and all of the transferred data appears to be used for the final calculation.

* Steps to Reproduce

Consider the simple example below. Test is ran for 2s and the first second should be omitted. I've marked issues with `!`.

    > iperf3 -c localhost -t 2 -O 1
    [ ID] Interval           Transfer     Bitrate         Retr  Cwnd
    [  5]   0.00-1.00   sec  3.17 GBytes  27.2 Gbits/sec    0   1.56 MBytes       (omitted)
    [  5]   0.00-1.00   sec  3.10 GBytes  26.6 Gbits/sec    0   1.62 MBytes       !
    [  5]   1.00-2.00   sec  3.01 GBytes  25.9 Gbits/sec    0   1.81 MBytes       
    - - - - - - - - - - - - - - - - - - - - - - - - -
    [ ID] Interval           Transfer     Bitrate         Retr
    [  5]  !0.00-2.00   sec !6.11 GBytes  26.3 Gbits/sec    0             sender
    [  5]  !0.00-2.00   sec !6.11 GBytes  26.3 Gbits/sec                  receiver

There are 5 `!` marks in the code. The first one corresponds to the probably unwanted line, while the others refer to the `0.0s-2.0s` interval used for the final average calculation instead of `1.0s-2.0s` one would expect when using `-t 2 -O 1`.

The first `!` in the raw output also shows that indentation is printed regardless of "`(omitted)`", which could also be cleaned up a bit when this is tackled. The unnecessary indendation at the end of the line can be seen in all non-omitted runtime lines (like the one below the mark).

**Comment by davidBar-On** [comment]
The problem seem to be the documentation/ help message.  Per the `create_client_timers()` function the test duration is the **sum** of the `-t` and `-O` values.  I.e. the omit time is not part of the total test time.  In case the the test is limited by number of bytes or blocks to transmit (`-n` or `-k`), these limits are not checked during the omit time.

@bmah888, probably the help text for `-O` should be changed to something like "perform pre-test for n seconds before starting the test.  Omit the pre-test statistics".

**Comment by drws** [comment]
Oh, I see. Thank you for information. It could also be called _a warmup_ or _a prewarm_ period.

**Comment by davidBar-On** [comment]
Created PR #1299 with suggested enhancement to the `--omit` help text.
